starter-spring/src/main/java/io/micrometer/prometheus/rsocket/autoconfigure/PrometheusRSocketClientProperties.java

```diff
@@ -63,6 +63,11 @@ public class PrometheusRSocketClientProperties {
    */
   private boolean secure = false;
 
+  /**
+   * The timeout in seconds to be used for establishing the connection and pushing the data
+   */
+  private long timeout = 5L;
```

Comment:
What do you think about using a `Duration` instead of `long`? Spring Boot has support for converting Durations (see [this section of the docs](https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.typesafe-configuration-properties.conversion.durations)).

Reply:
Good idea - I change it.

Reply:
Ok, the PR is updated. I didn't saw that you already used `Duration` for the retry configuration as well. 😃